drivers/hmis/spec/models/hmis/ce/referral_ce_event_manager_spec.rb

```diff
@@ -247,39 +261,82 @@ def submit_current_step(submitted_values = {})
       end
     end
 
-    before(:each) do
```

Comment:
I think this diff is getting exaggerated by Github, these examples are just tabbed in by 2 but otherwise unchanged :)